New package: FastBEAST v0.1.1

JuliaRegistrator opened the pull request:
- Registering package: FastBEAST
- Repository: https://github.com/FastBEAST/FastBEAST.jl
- Created by: @JoshuaTetzner
- Version: v0.1.1
- Commit: cea8a23515aa3126d5887081fb5743bcee803464
- Reviewed by: @JoshuaTetzner
- Reference: https://github.com/FastBEAST/FastBEAST.jl/commit/cea8a23515aa3126d5887081fb5743bcee803464#commitcomment-144527917
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6a72471b8b2d02f1f422393ec95f9dcaf34b035ddf9ec25325af51bfa75f0d0b30e17c3430d2886f986fb5f3bb6fa2e9871e0c1c37030fbea9f87d46e60f24e2b67034b9835b02693e482648742ce0f9f8734c03ea6130cda6741b9ff848455a9b0c48be3dbdb44b830c5b87ec42bac7797790cc984a553ef9b37b4707b11666f0ac9c00d0a97ec579ea8039ca2aba7586258cdc9adf4e31e986d500f4d4fb87831d6b03ef7b9b046dfec3ef66794a7c2 -->

goerz commented:
[noblock] I noticed that in the `README`, the text links to the documentation are broken

JuliaRegistrator committed:
New package: FastBEAST v0.1.1

UUID: 3fccbda2-992b-41a0-8fae-2b833e1e73d8
Repo: https://github.com/FastBEAST/FastBEAST.jl.git
Tree: bb63f8a1923d49f82ed186e3af24274f37908ddb

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
